scripts/util.js

```diff
@@ -1,11 +1,18 @@
+const Settings = require('./settings.js');
+const readline = require('readline');
+
 var Util = function(){};
 
 Util.refreshConsole = function(){
-  // var lines = process.stdout.getWindowSize()[1];
-  // for(var i = 0; i < lines; i++) {
-  //     console.log('\r\n');
-  // }
   process.stdout.write('\033c');
 };
 
+Util.overwriteConsole = function(){
+  if (Settings.getInstance().properties['disableColors']) {
+    process.stdout.write('\033c');
+  } else {
+    readline.cursorTo(process.stdout, 0, 0);
+  }
+};
+
```

Comment:
It'd be worth it to have a split setting for sure. That way if their terminal doesn't work with colors but does with readline they can still benefit from their buffer not being overwritten. It's a legitimate usecase imo.